### ISSUE TITLE
Email id verification for new sign up users

### ISSUE THREAD

**Issue by SUCHITRAGIRI** [Manthan933/Manthan#212]
**Is your feature request related to a problem? Please describe.**
When I was exploring this project I have noticed that there were no restriction of email ID for sign up, so 1st I signed up with my own valid email ID then for testing purpose when I entered a dummy email ID which also accepted, strange right!
It should not be there.

**Describe the solution you'd like**
So, I want to add one feature for verifying the email ID so that it will secure and no one can access the website without valid credential. And whoever want to sign up, on that time they have to verified their email ID via verification link which will be sent on their email ID. 

**Additional context**
![Screenshot (179)](https://user-images.githubusercontent.com/60110218/118477507-5fe1a700-b72c-11eb-8437-5ac871726fed.png)
![Screenshot (181)](https://user-images.githubusercontent.com/60110218/118477843-c6ff5b80-b72c-11eb-810a-098e026e531a.png)

See these emails are not exits, or I am not owned it but still I can use to access the website this means anyone can access without proper credential IDs

**Comment by SUCHITRAGIRI** [comment]
@JadeMaveric @Kathuria @atulya2109 @PuneetPunamiya  
Assign me this issue so that I can work on it:)
Thanks!

**Comment by SUCHITRAGIRI** [comment]
@aavishkarmishra can you help me out about docker set up actually I am very new to docker,  I am facing difficulty:(
Actually I have installed docker on my machine(windows) but wsl2 is not downloading. I did my all possible thing and also I am trying from 2 days if you can help it would  be better!

**Comment by SUCHITRAGIRI** [comment]
Yeah! I have already done via this and run the project on my local machine then I noticed in README about docker, so I had to move into it:(
So, I want to tell you that if here docker is not used then we should update the documentation part because It will mislead people like me!
And one thing if you want I can update the README, just let me :)

**Comment by adysis14** [comment]
Hey! I'm interested in solving this issue . I would be very grateful if you assign this issue to me under GSSOC'21

Thank You!

**Comment by vishutiwari99** [comment]
Hello Everyone, This is Vaibhav Tiwari. And I am just starting open source and want to contribute to this project. Can anyone assign me this issue to solve?

**Comment by aavishkarmishra** [comment]
Sure, please continue with this issue @vishutiwari99

**Comment by vishutiwari99** [comment]
Thanks @aavishkarmishra

**Comment by addymistrel** [comment]
Hello, I found this issue is opened from a very long time. It would be wonderful if you assign me this issue so that I can resolve it ASAP.
